src/data/polynomial/basic.lean

```diff
@@ -555,6 +555,17 @@ begin
   exact finsupp.sum_smul_index hf,
 end
 
+/-- `polynomial.sum` as a linear map. -/
+@[simps] def lsum {M : Type*} [add_comm_monoid M] [module R M] (f : ℕ → R →ₗ[R] M) :
+  polynomial R →ₗ[R] M :=
```

Comment:
I suspect this generalizes to
```suggestion
@[simps] def lsum {M : Type*} [add_comm_monoid M] [module R M] (f : ℕ → R →ₗ[S] M) :
  polynomial R →ₗ[S] M :=
```
given appropriate typeclasses

Reply:
thx, done